src/korone/tests/test_user_manager.py

```diff
@@ -0,0 +1,51 @@
+"""Korone is a simple multipurpose Telegram Bot.
+This is a complete rewrite of PyKorone, refer to
+https://github.com/AmanoTeam/PyKorone and README.md for more
+information."""
+
+from faker import Faker
+from pyrogram.types import User
+
+from korone.database import Database
+from korone.database.user import UserManager
+
+from pytest import fixture
+
+db = Database("korone-test.db")
```

Comment:
There has been a change in the Database API. You should easily be able to apply the changes for the test.